client/components/telecom/telephony/timeCondition/slot/telephony-time-condition-slot.html

```diff
@@ -21,10 +15,18 @@
 
         <div data-ng-if="!$ctrl.slot.noConfig">
             <!-- SLOT NOT DEFINED -->
-            <span data-ng-if="!$ctrl.getSlotDetail('number')"
-                  data-translate="voip_time_condition_slots_undefined"
-                  class="legend-description small font-weight-normal">
-            </span> <!-- END OF SLOT NOT DEFINED -->
+            <div data-ng-if="!$ctrl.getSlotDetail('number')">
+                <button class="oui-button_link p-0"
```

Comment:
No `.oui-button` ?

Reply:
No, because I have to  play with the padding, what cannot be done with classic `oui-button`